src/pkg/output/format.go

```diff
@@ -37,9 +38,17 @@ func AsYaml(v any) (string, error) {
 }
 
 func AsTable[T any](dataList []T, columns []string, getColumnData func(T) []map[string]string) (string, error) {
-	tab := tabulate.New(tabulate.SimpleUnicode)
-	for _, column := range columns {
-		tab.Header(column)
+	tab := tabulate.New(tabulate.Plain)
+	tab.Padding = 1
+	if len(dataList) == 0 {
+		return "", fmt.Errorf("no resources found")
```

Comment:
it shouldn't be considered an error. It's a valid case, and it should just return the error message to be printed as is - i.e. `return "No resources found", nil`. 
Also mind the capitalization.

Reply:
also - move this if to before you initialize `tab`.